Bottom: d368ffa61d4f438c63152f377eae210092a99150
Top:    c4ac103cf102ea870c8007fd16640b6d06933f5b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 19:39:44 +0000

Refresh of add-flag-parameters-to-resetfo

---

diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index 52479ff..af2ca1c 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -535,26 +535,18 @@ netlist_select_cb (GtkWidget * widget, gpointer data)
   if (selected_net == node_selected_net)
     node_selected_net = NULL;
 
+  SaveFindFlag (FOUNDFLAG);
   InitConnectionLookup ();
-  //ResetConnections (true);
-  {
-    extern int TheFlag;
-    printf ("gui-netlist-window.c: THE FLAG IS %i\n", TheFlag);
-    ResetConnections (true, TheFlag);
-  }
+  ResetConnections (true);
 
   for (i = selected_net->EntryN, entry = selected_net->Entry; i; i--, entry++)
     if (SeekPad (entry, &conn, false))
       RatFindHook (conn.type, conn.ptr1, conn.ptr2, conn.ptr2, true, true);
 
   SelectConnection (select_flag);
-  //ResetConnections (false);
-  {
-    extern int TheFlag;
-    printf ("gui-netlist-window.c: THE FLAG IS %i\n", TheFlag);
-    ResetConnections (false, TheFlag);
-  }
+  ResetConnections (false);
   FreeConnectionLookupMemory ();
+  RestoreFindFlag ();
   IncrementUndoSerialNumber ();
   Draw ();
 }
diff --git a/src/hid/lesstif/netlist.c b/src/hid/lesstif/netlist.c
index ba2cd4b..c53a9b7 100644
--- a/src/hid/lesstif/netlist.c
+++ b/src/hid/lesstif/netlist.c
@@ -132,6 +132,7 @@ nbcb_select_common (LibraryMenuType *net, int pos, int select_flag)
   ConnectionType conn;
   int i;
 
+  SaveFindFlag (FOUND);
   InitConnectionLookup ();
   ResetConnections (true);
 
@@ -142,6 +143,7 @@ nbcb_select_common (LibraryMenuType *net, int pos, int select_flag)
   SelectConnection (select_flag);
   ResetConnections (false);
   FreeConnectionLookupMemory ();
+  RestoreFindFlag ();
   IncrementUndoSerialNumber ();
   Draw ();
 }
